### DISCUSSION TITLE
请求新功能 去掉 搜索 评论 转发之类的输入框中的占位内容

### DISCUSSION THREAD

**Discussion by xx146542xx** [the1812/Bilibili-Evolved#3520] [功能建议 / Ideas]
![image](https://user-images.githubusercontent.com/109429390/180655419-88594964-ea91-433a-90a4-391782d3ade1.png)
![image](https://user-images.githubusercontent.com/109429390/180655435-9bd6548c-f8b2-4350-82dc-f3508edc14f2.png)
![image](https://user-images.githubusercontent.com/109429390/180655445-bbb02fa2-7685-42fc-a071-2c62b53b6089.png)
![image](https://user-images.githubusercontent.com/109429390/180655466-6abfdc54-7602-4b4c-9a10-b279a8b44792.png)
就这里面那些字 特别是搜索推荐 一天到晚推一些不感兴趣的甚至反感的内容

**Comment by the1812** [comment]
你是不是装了这个, 默认没有推荐的
![image](https://user-images.githubusercontent.com/26504152/180655573-5ed22a59-35a8-498e-8c4b-a01501a7425b.png)

**Reply by xx146542xx** [reply]
解决了
